kyo-prelude/shared/src/test/scala/kyo/AspectTest.scala

```diff
@@ -5,42 +5,30 @@ import kyo.Aspect.Cut
 class AspectTest extends Test:
 
     "one aspect" - {
-        val aspect       = Aspect.init[Int, Int, Any]
+        val aspect       = Aspect.init[Const[Int], Const[Int], Any]
         def test(v: Int) = aspect(v)(_ + 1)
 
         "default" in run {
             test(1).map(v => assert(v == 2))
         }
 
         "with cut" in run {
-            val cut = new Cut[Int, Int, Any]:
-                def apply(v: Int)(f: Int => Int < Any) =
-                    f(v + 1)
-            aspect.let[Assertion, Any](cut) {
+            aspect.let([C] => (input, cont) => cont(input + 1)) {
```

Comment:
More lightweight way to define cuts with `Cut.Binding`. I hope Scala will eventually allow omitting [C].